classes/userGroup/UserGroup.php

```diff
@@ -242,15 +244,10 @@ public function getPermitMetadataEdit()
 
     /**
      * Setter for permitMetadataEdit attribute.
-     *
-     * @param bool $permitMetadataEdit
      */
-    public function setPermitMetadataEdit($permitMetadataEdit)
+    public function setPermitMetadataEdit(?bool $permitMetadataEdit)
     {
-        $this->setData('permitMetadataEdit', $permitMetadataEdit);
+        if (isset($permitMetadataEdit))
+            $this->setData('permitMetadataEdit', $permitMetadataEdit);
     }
-}
-
-if (!PKP_STRICT_MODE) {
-    class_alias('\PKP\security\UserGroup', '\UserGroup');
```

Comment:
I'd recommend keeping the `class_alias` call for the sake of legacy plugins.

Reply:
Added this

```
if (!PKP_STRICT_MODE) {
    class_alias('\PKP\security\UserGroup', '\UserGroup');
}
```

at the end of the class after the closing `}`